editoast/src/client/mod.rs

```diff
@@ -143,6 +150,15 @@ pub struct ImportProfileSetArgs {
     pub electrical_profile_set_path: PathBuf,
 }
 
+#[derive(Args, Debug, Clone)]
+#[command(about, long_about = "Clone an infrastructure")]
+pub struct InfraCloneArgs {
+    /// Infrastructure ID
+    pub id: i64,
+    /// Infrastructure new name
+    pub new_name: String,
```

Comment:
This could be optional. 

If not provided use the original infra name and add a suffix like `[original_name] (copy)`.